A/ArcadeLearningEnvironment/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ArcadeLearningEnvironment"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/mgbellemare/Arcade-Learning-Environment/archive/v0.6.1.tar.gz", "8059a4087680da03878c1648a8ceb0413a341032ecaa44bef4ef1f9f829b6dde"),
+    ArchiveSource("https://zlib.net/zlib-1.2.11.tar.gz", "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1")
```

Comment:
We already have Zlib packaged and it can be used as dependency below:
```julia
dependencies = [
    Dependency("Zlib_jll"),
]
```
And remove this source
```suggestion
```

Reply:
I tried with Zlib_jll, but I guess I need some source files, since I always got the error
```
zlib.h: No such file or directory
```
until I figured out that I can use the flag `-DCMAKE_CXX_FLAGS=-isystem\ ../zlib-1.2.11/`.
Is there a better alternative?

Reply:
Yes, reading the documentation: https://juliapackaging.github.io/BinaryBuilder.jl/dev/troubleshooting/#Header-files-of-the-dependencies-can't-be-found-1 :wink:

Reply:
Ah, my goodness. Sorry and thanks a lot!